custom_components/tuya_local/devices/kilo_plus_air_purifier.yaml

```diff
@@ -1,4 +1,4 @@
-name: Air purifier
+name: Air Purifier
```

Comment:
This change is incorrect.

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/

Reply:
You are correct. Thanks for all this info, this is my first HA related PR.